pkgs/development/python-modules/gb-io/default.nix

```
@@ -0,0 +1,31 @@
+{ fetchFromGitHub
+, buildPythonPackage
+, rustPlatform
+, setuptools-rust
+}:
+
+buildPythonPackage rec {
+  pname = "gb-io";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "althonos";
+    repo = "gb-io.py";
+    rev = "v${version}";
+    sha256 = "0s0xm9x8lbcs7pw4bgxlqqhisgrs6g2a37bqkg91hjcz1qis1hl2";
+  };
+
+  cargoDeps = rustPlatform.fetchCargoTarball {
+    inherit src sourceRoot;
+    name = "${pname}-${version}";
```

Bit nicer to read i think:
```suggestion
    name = "gb-io-${version}";
```